src/Files.App/Views/SettingsPages/Tags.xaml

```diff
@@ -221,8 +221,34 @@
 												Padding="8,4"
 												Background="Transparent"
 												BorderBrush="Transparent"
-												Click="RemoveTag_Click"
-												Content="{helpers:ResourceString Name=Delete}" />
+												Click="PreRemoveTag_Click"
+												Content="{helpers:ResourceString Name=Delete}">
+												<Button.Flyout>
+													<Flyout>
+														<StackPanel
+															MaxWidth="150"
+															Padding="0"
```

Comment:
Isn't padding `0` by default?

Reply:
Oh, you're right. I don't know why I was thinking it was `3`